src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -2402,19 +2375,21 @@ public async Task<JArray> GetObjectValues(int objectId, GetObjectCommandOptions
             {
                 var retAfterRemove = new JArray();
                 List<List<FieldTypeClass>> allFields = new List<List<FieldTypeClass>>();
-                for (int i = 0; i < typeId.Count; i++)
+                for (int i = 0; i < typeIdsIncludingParent.Count; i++)
                 {
-                    var fields = await GetTypeFields(typeId[i], token);
-                    allFields.Add(fields);
+                    var fields = await GetTypeFields(typeIdsIncludingParent[i], token);
+                    var (regularFields, _) = await FilterFieldsByDebuggerBrowsable(fields, typeIdsIncludingParent[i], token);
```

Comment:
Why only `regularFields` in this case?

Reply:
Because I did not fully understand what we are doing here and that it's a step to get rid of Accessor properties that already have their corresponding field defined :D 
Ofc you're right and we should accumulate both in *allFields*.